test/test_torch.py

```diff
@@ -15458,6 +15458,13 @@ def test_abs_signed_zero(self, device, dtype):
         for v in inp:
             self.assertGreater(math.copysign(1.0, v), 0.0)
 
+    @dtypes(torch.float)
+    def test_absolute(self, device, dtype):
+        # absolute is an alias for abs. Just check to see that results
+        # are the same.
+        t = torch.randn(10, 10, device=device, dtype=dtype)
+        self.assertEqual(t.abs(), t.absolute())
```

Comment:
I neglected to mention this earlier, but let's test the variants (out=, t.absolute(), and t.absolute_) too